src/logic/showQuestion.js

```diff
@@ -0,0 +1,13 @@
+
+function showQuestion(nr) {
+    // Hide the setup
+    document.getElementById("explanation_div").style.display = "none";
+    // Show the game
+    document.getElementById("play_quiz_div").style.display = "block";
```

Comment:
in `init.js` these eventlistener codes already exist, I just wonder why you wrote it again ?
This branch works good, i am approving. Good work  💯 @aleks2407

Reply:
@oguzkarademir,in init.js we display the first DIV with the START button and hide the DIV with the question and rest of the buttons
Here we hide the DIV with START button and display the DIV with the question and rest of the buttons
Look more carefully on "block" and "none" , they are switched

Reply:
> @oguzkarademir,in init.js we display the first DIV with the START button and hide the DIV with the question and rest of the buttons
> Here we hide the DIV with START button and display the DIV with the question and rest of the buttons
> Look more carefully on "block" and "none" , they are switched

i understood,  thanks